src/lib/firebase.js

```diff
@@ -1,18 +1,17 @@
 // NOTE: import only the Firebase modules that you need in your app... except
 // for the second line, which makes both the linter and react-firebase happy
-import firebase from 'firebase/app';
-import 'firebase/firestore';
+import firebase from 'firebase';
 
 // Initalize Firebase.
 var firebaseConfig = {
-  apiKey: "AIzaSyB1pamXP9wq65aAUlJuS1P7NRWPGi5FJ5g",
-  authDomain: "tcl-29-shopping-list.firebaseapp.com",
-  projectId: "tcl-29-shopping-list",
-  storageBucket: "tcl-29-shopping-list.appspot.com",
-  messagingSenderId: "166083757710",
-  appId: "1:166083757710:web:d959706ebc407afbc1abdf"
+  apiKey: 'AIzaSyB1pamXP9wq65aAUlJuS1P7NRWPGi5FJ5g',
+  authDomain: 'tcl-29-shopping-list.firebaseapp.com',
+  projectId: 'tcl-29-shopping-list',
+  storageBucket: 'tcl-29-shopping-list.appspot.com',
+  messagingSenderId: '166083757710',
+  appId: '1:166083757710:web:d959706ebc407afbc1abdf',
 };
 
 let fb = firebase.initializeApp(firebaseConfig);
-
-export { fb };
+const db = fb.firestore();
```

Comment:
Ooh this looks interesting. How did you figure it out that we can export firestore directly here?

Reply:
This was something @ishimwezachee realised by looking at the documentation! It looks a lot cleaner so we implemented it :)

Reply:
Disclaimer: I'm not familiar with the _"right"_ way to do this, but it looks like the changes in commit f120ef2d315817ece8a7bd33c5ce4f53e44f706d introduced a warning in the console.

Check out the [preview](https://tcl-29-shopping-list--pr21-zi-an-add-new-item-6pmskj1u.web.app/) and open the dev tools console. You'll see an error that is suggesting to me that while it's not technically broken, we're circumventing something. From what I can tell, it's informing us that we're basically importing all of firebase instead of just importing what we need, resulting in a much larger app.

<img width="350px" alt="err" src="https://user-images.githubusercontent.com/8881164/126775936-b3f23e2f-05e0-4bc0-b39d-eaf42e4f3e45.png">